### PR TITLE
Switch to CommonJS export for module: 'node16'

### PR DESCRIPTION
**Pull request by karlhorky**
As per https://github.com/Kikobeats/free-email-domains/pull/37#issuecomment-1410885100

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 declare module 'is-url-http' {
-  export default function isUrlHttp(url: string): boolean;
-}
\ No newline at end of file
+  export = function isUrlHttp(url: string): boolean;
```
Comment (andrewbranch) [inline index.d.ts]: For weird syntax reasons, it’s not quite inlinable like that
```suggestion
  function isUrlHttp(url: string): boolean;
  export = isUrlHttp;
```

**Review by andrewbranch — CHANGES_REQUESTED** [review]
I would also remove the `declare module` wrapper as you suggested, which means the `declare` modifier will move to before `function`.

**Comment by karlhorky** [comment]
Done, removed the ambient declaration, thanks for the guidance!

**Review by andrewbranch — APPROVED** [review]

**Comment by Kikobeats** [comment]
Thanks!

**Comment by karlhorky** [comment]
Glad to help, thanks for the merge!

**Pull request merged**